mailpoet/tests/integration/Mailer/Methods/AmazonSESTest.php

```diff
@@ -255,4 +258,33 @@ public function testItChecksBlacklistBeforeSending() {
     expect($result['error'])->isInstanceOf(MailerError::class);
     expect($result['error']->getMessage())->stringContainsString('AmazonSES has returned an unknown error.');
   }
+
+  public function testItCanSend() {
+    if (getenv('WP_TEST_MAILER_ENABLE_SENDING') !== 'true') $this->markTestSkipped();
```

Comment:
I think that we don't need to skip this test anymore when the sending is mocked. What do you think?

Reply:
@lysyjan umm, yea makes sense, updating.